test/alternator/test_metrics.py

```diff
@@ -439,6 +485,60 @@ def test_streams_latency(dynamodb, dynamodbstreams, metrics):
         with check_sets_latency(metrics, ['GetRecords']):
             dynamodbstreams.get_records(ShardIterator=it)
 
+###### Test metrics that are item size histograms of DynamoDB API operations which manipulate data:
+# Tests for histogram metrics operation_size_kib.
+
+INF = float('inf')
+
+def check_histogram_metric_increases(op, name, metrics, do_test, probes):
+    points = [[value, {'op': op, 'le': le}] for value, le in probes]
+    metric_bucket = f'scylla_alternator_table_{name}_bucket'
+    with check_increases_metric_exact(metrics, metric_bucket, points):
+        do_test()
+
+@pytest.mark.parametrize("force_rbw", [True, False])
+def test_get_item_size_not_found_doesnt_increase_the_metric(dynamodb, test_table_s, metrics, force_rbw):
+    with scylla_config_temporary(dynamodb, 'alternator_force_read_before_write', str(force_rbw).lower()):
+        def do_test():
+            test_table_s.get_item(Key={'p': random_string()})
+        check_histogram_metric_increases('GetItem', 'operation_size_kib', metrics, do_test, [(0, bucket(float('inf')))])
```

Comment:
nitpick: you set INF above as an alias to float('inf'), but don't use it it here.